### DISCUSSION TITLE
Saturn 2 build volume

### DISCUSSION THREAD

**Discussion by adriangalilea** [sn4k3/UVtools#672] [Q&A] [answered]
Hello,

I was trying to use UVTools for the first time with my Elegoo Saturn 2, when I imported the profile I got this weird build volume:

<img width="575" alt="Screenshot 2023-03-04 at 17 01 32" src="https://user-images.githubusercontent.com/90320947/222916229-7f4fa41a-82e2-49ec-aa96-e9208f82fd7f.png">

But the actual build volume is: "Build Volume 219x123x250mm" according to google.

Is there any reason why the profile has those dimensions?

**Comment by sn4k3** [comment, marked as answer]
> Is there any reason why the profile has those dimensions?

Yes, most of the times announced specs it's rounded to be pretty but should never use. Never trust online specs, instead use math!
You know pixel size and resolution (exact values) then display is not exact so you can do math to find it.

Resolution: `7680 x 4320`
Pixel size:  `28.5 microns / 0,0285 mm`

Then: 
Display width= `0,0285 * 7680 = 218,88 mm`
Display height = `0,0285 * 4320 = 123,12 mm`

If you use the specs online you will get a pixel size of: `219 / 7680 = 0,028515625 mm` which already have a error.

Using the specs online will cause dimension error and in a long model can be noticed to be distorced. Trust UVtools ;)

**Reply by adriangalilea** [reply]
Dude you are amazing, thanks.

**Comment by adriangalilea** [comment]
Now that I realize chitubox agrees with the build volume and it never stood out to me 

<img width="1149" alt="image" src="https://user-images.githubusercontent.com/90320947/222926473-68f93f49-e132-4cc0-8612-76509bc76690.png">

Offtopic minute doubts(I can create another thread if you wish):
On material settings prusaslicer has an initial layer height, I got no idea what that would be different than any other layer height so I also specified 0.03 mm.

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/90320947/222926590-ef2b58fa-7a5c-4240-b938-661b640eef3d.png">
And I'm assuming I don't have to select any of these into PrusaSlicer somehow, right?

🙏

**Reply by sn4k3** [reply]
> On material settings prusaslicer has an initial layer height, I got no idea what that would be different than any other layer height so I also specified 0.03 mm.

That wont work for other printers. Keep as same as layer height.

> And I'm assuming I don't have to select any of these into PrusaSlicer somehow, right?


You should... They are on Printer tab, on Notes

**Reply by adriangalilea** [reply]
Oh!

Now I see them but I'm confused as to how to translate the values is there any documentation on this?

`LightOffDelay_0`
Can't find this on chitubox
`BottomLightOffDelay_0`
Can't find this on chitubox
`BottomLiftHeight_7`
Bottom Lift Distance I assume but I have:
`Bottom Lift Distance 3 + 4 mm`
so not sure how that translates, maybe 7 is right (?)
`LiftHeight_7`
Lifting distance I assume and 3+4 so might be 7 too
`BottomLiftSpeed_70`
I have 65 & 180 which I'm not sure how to translate
`LiftSpeed_70`
I have 45 & 190 which I'm not sure how to translate
`RetractSpeed_210`
I have 180 & 65 which I'm not sure how to translate

Thank you for all the replies 🙏

**Reply by sn4k3** [reply]
Read this: https://github.com/sn4k3/UVtools#prusaslicer
